ddp-workspace/projects/ddp-atcp/src/app/components/about-initiative/about-initiative.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { NGXTranslateService } from 'ddp-sdk';
+
+@Component({
+  selector: 'app-about-initiative',
+  templateUrl: './about-initiative.html',
+  styleUrls: ['./about-initiative.scss']
+})
+export class AboutInitiativeComponent implements OnInit {
+  public AboutInitiative;
+  public steps;
```

Comment:
Could you also specify data types for properties in all classes if they don't have default value?